src/store/peer/reducer.ts

```diff
@@ -2,15 +2,18 @@ import { createReducer } from 'typesafe-actions';
 import { combineReducers } from 'redux';
 import { Peer } from './types';
 import actions, { ActionTypes } from './actions';
-import {ApiError, DeleteResponse} from "../../services/api-client/types";
+import {ApiError, ChangeResponse, DeleteResponse} from "../../services/api-client/types";
+import {Group} from "../group/types";
```

Comment:
Codacy has a fix for the issue: [Replace `Group` with `·Group·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
import { Group } from "../group/types";
```